examples/jest-snapshot-testing/src/components/AppFooter.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import { FormattedMessage, defineMessages } from 'react-intl';
+
+const messages = defineMessages({
+    copyright: {
```

Comment:
There's a mix of 2 and 4 space indentation. If you don't mind can you make it consistent throughout the example?